jerry-core/ecma/operations/ecma-conversion.cpp

```diff
@@ -489,9 +489,6 @@ ecma_op_from_property_descriptor (const ecma_property_descriptor_t* src_prop_des
   else
   {
     // 4.
-    JERRY_ASSERT (src_prop_desc_p->is_get_defined
-                  || src_prop_desc_p->is_set_defined);
```

Comment:
Yes, it's valid solution.

This is not a hack, but the only right way to represent 'undefined' value of defined [[Get]] in ecma_property_descriptor_t, according to current design of the structure.

If you think that this should be implemented in another way, please describe the new design here or in a new issue, for discussion.